bricks/basic-bricks/src/resizable-box/index.tsx

```diff
@@ -38,6 +38,9 @@ export class ResizableBoxElement extends UpdatingElement {
   @property({ attribute: false })
   boxStyleWhenNotResizing: React.CSSProperties;
 
+  @property({ attribute: false })
+  barUiType: "dashboard" | "default" = "default";
```

Comment:
这个参数有点拗口，换这个吧？另外后面万一 dashboard 不止要改 bar 的 ui 呢？

```suggestion
  variant: "dashboard" | "default" = "default";
```

Reply:
好的，那我替换一下参数